README-nl.md

```diff
@@ -10,64 +10,64 @@ Minecraft Console Client (MCC) is een simpele app waarmee je Minecraft servers k
 Krijg het binary bestand van onze laatste [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest).
```

Comment:
"Verkrijg [hier](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest) de laatste ontwikkelde binaire versie.

Reply:
I just changed it and i think it is completely okay now.